plugins/task-assets/lang/en.json

```diff
@@ -29,6 +29,7 @@
     "InProgress": "In progress",
     "Closed": "Closed",
     "Favorite": "Favorite",
-    "MyTasks": "My tasks"
+    "MyTasks": "My tasks",
+    "PleaseSelectSpace": "Space selection is required."
```

Comment:
Maybe use a "project" for tasks? I'm not sure because we have specials

Reply:
I think we need to do something more interesting, like showing some dashboard etc. So let it be for now.